R-packages/covidcast/R/plot.R

```diff
@@ -8,8 +8,11 @@ plot_choro = function(x, time_value = NULL, include = c(), range,
                       title = NULL, params = list()) {
   # Check that we're looking at either counties or states
   if (!(attributes(x)$geo_type == "county" ||
-                     attributes(x)$geo_type == "state")) {
-    stop("Only 'county' and 'state' are supported for choropleth maps.")
+                     attributes(x)$geo_type == "state" ||
+                     attributes(x)$geo_type == "hrr" ||
+                     attributes(x)$geo_type == "msa")) {
```

Comment:
At this point we may as well do `attributes(x)$geo_type %in% c("county", "state", "hrr", "msa")`, since there are so many